src/Redux/book/book.js

```diff
@@ -0,0 +1,23 @@
+// Actions
+const ADD_BOOK = 'ADD_BOOK';
+const REMOVE_BOOK = 'REMOVE_BOOK';
```

Comment:
Please make the necessary changes so that the value of each action type needs to follow the [ducks pattern](https://github.com/erikras/ducks-modular-redux), so it will be something like: 
`const ADD_BOOK = bookstore/books/ADD_BOOK` 
and 
`const REMOVE_BOOK = bookstore/books/REMOVE_BOOK`

Reply:
Thanks a lot, I completely forgot and acctually didn't pay attention to that part. Thanks for pointing that out.